ui/src/app/pages/profile/profile-page.tsx

```diff
@@ -194,208 +319,237 @@ export const ProfilePage = withUserProfile()(class extends React.Component<
+                isComplete={!!(getRegistrationTasksMap()['eraCommons'].completionTimestamp(profile))}
+                completeStep={getRegistrationTasksMap()['eraCommons'].onClick} >
+              <div>
+                {profile.eraCommonsLinkedNihUsername != null && <React.Fragment>
+                  <div> Username: </div>
+                  <div> { profile.eraCommonsLinkedNihUsername } </div>
+                </React.Fragment>}
+                {profile.eraCommonsLinkExpireTime != null &&
+                //  Firecloud returns eraCommons link expiration as 0 if there is no linked account.
+                profile.eraCommonsLinkExpireTime !== 0
+                && <React.Fragment>
+                  <div> Link Expiration: </div>
+                  <div>
+                    { moment.unix(profile.eraCommonsLinkExpireTime)
+                        .format('MMMM Do, YYYY, h:mm:ss A') }
+                  </div>
+                </React.Fragment>}
+              </div>
+            </ProfileRegistrationStepStatus>}
+
+            {enableComplianceTraining && <ProfileRegistrationStepStatus
+                title='All of Us Responsible Conduct of Research Training'
+                wasBypassed={!!profile.complianceTrainingBypassTime}
+                incompleteButtonText='Access Training'
+                completedButtonText={getRegistrationTasksMap()['complianceTraining'].completedText}
+                completionTimestamp={getRegistrationTasksMap()['complianceTraining'].completionTimestamp(profile)}
+                isComplete={!!(getRegistrationTasksMap()['complianceTraining'].completionTimestamp(profile))}
+                completeStep={getRegistrationTasksMap()['complianceTraining'].onClick} />}
+
+
+            {enableDataUseAgreement && <ProfileRegistrationStepStatus
+                title='Sign Data User Code Of Conduct'
+                wasBypassed={!!profile.dataUseAgreementBypassTime}
+                incompleteButtonText='Sign'
+                completedButtonText={getRegistrationTasksMap()['dataUserCodeOfConduct'].completedText}
+                completionTimestamp={getRegistrationTasksMap()['dataUserCodeOfConduct'].completionTimestamp(profile)}
+                isComplete={!!(getRegistrationTasksMap()['dataUserCodeOfConduct'].completionTimestamp(profile))}
+                completeStep={getRegistrationTasksMap()['dataUserCodeOfConduct'].onClick} >
+              {profile.dataUseAgreementCompletionTime != null && <React.Fragment>
+                <div> Agreement Renewal: </div>
+                <div>
+                  { moment.unix(profile.dataUseAgreementCompletionTime / 1000)
+                      .add(1, 'year')
+                      .format('MMMM Do, YYYY') }
+                </div>
+              </React.Fragment>}
+              <a
+                  onClick={getRegistrationTasksMap()['dataUserCodeOfConduct'].onClick}>
+                View current agreement
+              </a>
```

Comment:
This 'View current agreement' looks out of place on the screenshots,  could you make sure it's left-aligned and not crowding the 'completed' button?